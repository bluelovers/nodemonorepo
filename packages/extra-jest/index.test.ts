import * as subject from './index'

it('matches snapshot', () => {
  expect(subject).toMatchSnapshot()
})
