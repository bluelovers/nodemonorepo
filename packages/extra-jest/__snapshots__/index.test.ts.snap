// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "default": Object {
    "snap": Object {
      "default": [Function],
      "mkfn": [Function],
      "noRefs": [Function],
      "pureSafe": [Function],
      "pureUnsafe": [Function],
      "safe": [Function],
      "unsafe": [Function],
    },
  },
  "snap": Object {
    "default": [Function],
    "mkfn": [Function],
    "noRefs": [Function],
    "pureSafe": [Function],
    "pureUnsafe": [Function],
    "safe": [Function],
    "unsafe": [Function],
  },
}
`;
